src/python_tools/notebooks/plot_ssl_wrapperpacket.ipynb

```diff
@@ -3,21 +3,23 @@
   {
    "cell_type": "code",
    "execution_count": 1,
-   "id": "later-plasma",
+   "id": "common-serbia",
```

Comment:
just wondering what this is

Reply:
its the output of bokeh plotting, this is not code that i changed or anything. notebooks will become easier to review after #1910 is resolved.